src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/VideoLinkBookingService.kt

```diff
@@ -339,4 +343,27 @@ class VideoLinkBookingService(
       videoLinkBookingEventListener.appointmentUpdatedInNomis(videoLinkAppointment, appointmentChangedEventMessage)
     }
   }
+
+  fun deleteAppointments(releasedOffenderEventMessage: ReleasedOffenderEventMessage) {
```

Comment:
might be better to clarify what this method is doing i.e. renamed to `maybeDeleteAppointments` since it doesn't always delete the appointments, or `deleteAppointmentsIfNoLongerInPrison`